source/util/ilist.h

```diff
@@ -179,37 +183,37 @@ IntrusiveList<NodeType>& IntrusiveList<NodeType>::operator=(
 
 template <class NodeType>
 inline typename IntrusiveList<NodeType>::iterator
-IntrusiveList<NodeType>::begin() noexcept {
+IntrusiveList<NodeType>::begin() {
   return iterator(sentinel_.next_node_);
 }
 
 template <class NodeType>
 inline typename IntrusiveList<NodeType>::iterator
-IntrusiveList<NodeType>::end() noexcept {
+IntrusiveList<NodeType>::end() {
   return iterator(&sentinel_);
 }
 
 template <class NodeType>
 inline typename IntrusiveList<NodeType>::const_iterator
-IntrusiveList<NodeType>::begin() const noexcept {
+IntrusiveList<NodeType>::begin() const {
   return const_iterator(sentinel_.next_node_);
 }
 
 template <class NodeType>
 inline typename IntrusiveList<NodeType>::const_iterator
-IntrusiveList<NodeType>::end() const noexcept {
+IntrusiveList<NodeType>::end() const {
   return const_iterator(&sentinel_);
 }
 
 template <class NodeType>
 inline typename IntrusiveList<NodeType>::const_iterator
-IntrusiveList<NodeType>::cbegin() const noexcept {
+IntrusiveList<NodeType>::cbegin() const {
   return const_iterator(sentinel_.next_node_);
 }
 
 template <class NodeType>
 inline typename IntrusiveList<NodeType>::const_iterator
-IntrusiveList<NodeType>::cend() const noexcept {
```

Comment:
You could still have it as NOEXCEPT which would be nothing for MSVC under 2015. But I don't insist, it was David's idea to use noexcept in the first place.

Reply:
Ok to leave out.  :-)

Reply:
We compile without exceptions anyway.